flake.nix

```diff
@@ -20,8 +20,12 @@
     ixset-typed.flake = false;
     tailwind-haskell.url = "github:srid/tailwind-haskell/master";
     tailwind-haskell.inputs.nixpkgs.follows = "nixpkgs";
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
```

Comment:
We don't need to add the `flake-compat` input here, as it can be referenced directly from `default.nix`. See https://github.com/srid/emanote/blob/master/shell.nix

Reply:
Thank you for your reply! And I apologize for adding the files back. I don't mind having my own fork with this patch! I don't think my decision to keep with legacy nix should be imposed in the repository!

Reply:
Oh, no worries. I'm still open to adding the old `default.nix` back (but without proxying to the flake input because, as the current shell.nix shows, we don't need that).

Reply:
I see, thanks I'll edit then!